google/cloud/security/common/data_access/sql_queries/select_data.py

```diff
@@ -65,10 +65,16 @@
     SELECT group_id from groups_{0};
 """
 
-GROUP_USERS = """
+GROUP_ID = """
+    SELECT group_id
+    FROM groups_{1}
+    WHERE group_email = "{0}";
+"""
+
+GROUP_MEMBERS = """
     SELECT group_id, member_role, member_type, member_id, member_email
-    FROM group_members_{0}
-    WHERE member_status = "ACTIVE" and member_type = "USER";
+    FROM group_members_{1}
+    WHERE group_id = "{0}";
```

Comment:
Hmmm, I am not sure I understand.  Can you give me an example?
One my part, this is how I'm currently formatting the sql query.  So, positionally, I need to explicitly use {0} for group_id, and {2} for the timestamp.
```
sql = select_data.GROUP_MEMBERS.format(group_id, timestamp)
```